### PR TITLE
Improve pkg/kubeapiserver/options/options.go

### PR DESCRIPTION
**Pull request by SataQiu**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
Improve pkg/kubeapiserver/options/options.go:
- Block definition
- Define more common variables/constants

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/kube-apiserver/app/server.go [hunk 1]
```diff
@@ -430,7 +430,7 @@ func buildGenericConfig(
 	// Since not every generic apiserver has to support protobufs, we
 	// cannot default to it in generic apiserver and need to explicitly
 	// set it in kube-apiserver.
-	genericConfig.LoopbackClientConfig.ContentConfig.ContentType = "application/vnd.kubernetes.protobuf"
+	genericConfig.LoopbackClientConfig.ContentConfig.ContentType = kubeoptions.DefaultStorageMediaType
```
Comment (yue9944882) [inline cmd/kube-apiserver/app/server.go]: it messes up api content type and storage content type

Comment (SataQiu) [reply cmd/kube-apiserver/app/server.go]: Thanks a lot! Updated.

**Pull request closed without merging**